packages/design-system/src/scalars/components/radio-group/radio/radio.test.tsx

```diff
@@ -0,0 +1,87 @@
+import { describe, expect, it } from "vitest";
+import { Radio } from "../radio";
+import { RadioGroup } from "../radio-group";
+import { render, screen } from "@testing-library/react";
+
+describe("Radio Component", () => {
+  it("should match snapshot", () => {
+    const { asFragment } = render(
+      <RadioGroup>
+        <Radio label="Test Label" value="test" />
+      </RadioGroup>,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it("renders with label and value", () => {
```

Comment:
by convention, each test should start with the word "should". Eg: "should render with label and value"

Reply:
Done